.github/workflows/deploy.yaml

```diff
@@ -224,7 +226,7 @@ jobs:
       # Use rsync to deploy the release apk on the website
       - name: Sync
         env:
-            dest: "popdemo@${{secrets.POPDEMO_DEPLOY_ADDR}}:fe2-android/"
+            dest: "popdemo@${{secrets.POPDEMO_DEPLOY_ADDR}}:dev/fe2-android/"
```

Comment:
this is on purpose. fe2 is always in the "dev" environment.
it just contains all the APKs (dev and prod alike)